mitmproxy/contentviews/__init__.py

```diff
@@ -101,7 +101,7 @@ def get_message_content_view(
         http_message = message
         if ctype := message.headers.get("content-type"):
             if ct := http.parse_content_type(ctype):
-                content_type = f"{ct[0]}/{ct[1]}"
+                content_type = http.assemble_content_type(ct[0], ct[1], ct[2])
```

Comment:
I believe we never need the content type attributes when selecting a contentview? Unless there's something I am missing, please revert the changes here.